_docs/Popup/popup.md

```diff
@@ -0,0 +1,42 @@
+# Popup Component
+
+**Popup** is a component that pops up content and positions it next to an anchor element. A Popup is overlayed over the page. A Popup is a temporary view and only one popup can be rendered on the page at one time.
```

Comment:
@giladgrush can you describe the specific use-case for what we're calling `PopUp`?

@idoros / @giladgrush / @LeonidLevi let's get the nuances between popup, modal, dialog, portal and whatever else i'm missing written down